device-scripts/diag.sh

```diff
@@ -0,0 +1,186 @@
+#!/bin/sh
+#
+# Copyright (C) 2023 Foundries.io
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Suggested setting with vim:
+# set ai ts=4 sw=4 et
+#
+# Authors:
+#    Tim Anderson <tim.anderson@foundries.io>
+#    Vanessa Maegima <vanessa.maegima@foundries.io>
+#    Caio Pereira <caio.pereira@foundries.io>
+
+VERSION=1.0
+
+echo "*** diag tool version ***"
+echo "Version: $VERSION"
+echo
+
+echo "*** os-release ***"
+cat /etc/os-release
+echo
+
+test -f /var/sota/current-target && echo -e "*** current_target ***\n$(cat /var/sota/current-target)"
+echo
+
+echo "*** active networks ***"
+for chk in $(ip addr | grep "^[0-9]" | sed 's/^[0-9]*://;s/:.*//')
+do
+    ifconfig $chk 2> /dev/null | grep -q inet
+    if [ $? -eq 0 ]
+     then
```

Comment:
Extra space here (I would personally have do/then all at the same for/while/if line, but that is just me).